keps/sig-auth/3257-clustertrustbundles/README.md

```diff
@@ -342,11 +346,21 @@ ClusterTrustBundle objects may optionally be associated with a signer using the
 additional admission control and access control for the ClusterTrustBundle
```

Comment:
Above we say:

> Security: Should individual trust anchor set entries designate an OSCP endpoint to check for certificate revociation? Or should we require the URL to be embedded in the issued certificates? Note: This question has been deferred from the 1.26 alpha scope.

I believe we need to address this now.

Reply:
I think tactically we are going to have to punt this decision to 1.31, since we're about to hit the enhancements freeze.  It would be nice to get the API types themselves to beta in 1.30.